src/App.tsx

```diff
@@ -1,61 +1,147 @@
 import './App.scss';
+import {
+  Box,
+  Paper,
+} from '@mui/material';
```

Comment:
Include `package.json` and `package-lock.json` to the PR to make it work

![Screenshot 2023-04-05 at 19 03 33](https://user-images.githubusercontent.com/54940595/230140469-335aac64-d59f-42d9-9758-2fd7ac0c42c7.png)